scipy/spatial/transform/rotation.py

```diff
@@ -3,12 +3,139 @@
 import numpy as np
 import scipy.linalg
 import re
+import warnings
 
 
 AXIS_TO_IND = {'x': 0, 'y': 1, 'z': 2}
 
 
-def _make_elementary_quat(axis, angles):
+def elementary_basis_vector(axis):
```

Comment:
Anything that we do not to support the user using (e.g., no docstring, not in docs, etc.) should be preceded by a leading `_` to make it private rather than public -- that is a signal to users they should not expect API stability if they go browsing the `scipy.spatial.transform.rotation` namespace. In general you should restrict the set of public functions as much as possible to reduce maintenance burden and keep the code as flexible as possible for future updates.